pkg/controllers/dashboard/helm/repo.go

```diff
@@ -247,6 +255,23 @@ func (r *repoHandler) download(repoSpec *catalog.RepoSpec, status catalog.RepoSt
 	return status, nil
 }
 
+func (r *repoHandler) checkConfigMap(repo *catalog.ClusterRepo, status *catalog.RepoStatus) error {
```

Comment:
Suggesting to change func name to something that relays the fact that status may get changed. `ensureConfigMap` maybe?

Reply:
the name is updated to `ensureIndexConfigMap`